tests/src/LuaTests.cs

```diff
@@ -1766,6 +1766,111 @@ public void TestPrintChars()
             }
         }
 
+        [Test]
+        public void TestPrintNoParams()
+        {
+            using (Lua lua = new Lua())
+            {
+                lua.DoString(@"print()");
+                Assert.IsTrue(true);
+            }
+        }
+
+        [Test]
+        public void TestPrintSingleNil()
+        {
+            using (Lua lua = new Lua())
+            {
+                lua.DoString(@"print(nil)");
+                Assert.IsTrue(true);
```

Comment:
One more thing. 
What if we make Print retur the string and use the return in the assertion instead always true?
This way you could compare the expected result with the actual result avoiding future regressions

Reply:
I've made Print return the string and we now compare the first chunk returned from Lua with the expected result. Is this what you meant + is it now in a way you'd like to see it?

Thanks for the feedback, learning a lot!